components/PageCompanies.tsx

```diff
@@ -0,0 +1,122 @@
+import Head from "next/head"
+import { PropsWithChildren } from "react"
+import styled from "styled-components"
+
+import { EnvVars } from "env"
+import { media } from "utils/media"
+
+import Container from "./Container"
+import SectionTitle from "./SectionTitle"
+
+import Button from "./Button"
+import Link from "./Link"
+
+
+
+
+export interface PageProps {
```

Comment:
The nav props can be passed as an array of objects.

Reply:
When you'll pass an array you'll simply expand them by `array.map` this would also ensure that you don't need two different components `PageStudent` and `PageCompany`.

There must be only one `Page` Component